deps/obs-scripting/obs-scripting-python-import.c

```diff
@@ -65,9 +65,9 @@ bool import_python(const char *python_path)
 
 	char temp[PATH_MAX];
 
-	sprintf(cur_version, VERSION_PATTERN, PY_MAJOR_VERSION_MAX,
-		PY_MINOR_VERSION_MAX);
-	sprintf(temp, FILE_PATTERN, cur_version);
+	snprintf(cur_version, 5, VERSION_PATTERN, PY_MAJOR_VERSION_MAX,
```

Comment:
Should be `sizeof(cur_version)`, using literals risks errors when one is changed but not the other.